test/format.test.ts

```diff
@@ -180,3 +180,27 @@ describe("xlsx-wasm test", () => {
     expect(actual).matchXlsx(expected);
   });
 });
+
+describe("xlsx-wasm test", () => {
+test("issue6", async () => {
```

Comment:
Would you mind using a clearer name so that others can easily understand what is being tested?

Reply:
Thanks. fixed